stdlib/2/os/path.pyi

```diff
@@ -124,28 +115,28 @@ if sys.version_info < (3, 0):
     @overload
     def join(__p1: bytes, *p: bytes) -> bytes: ...
     @overload
-    def join(__p1: bytes, __p2: bytes, __p3: bytes, __p4: Text, *p: _PathType) -> Text: ...
+    def join(__p1: bytes, __p2: bytes, __p3: bytes, __p4: Text, *p: AnyPath) -> Text: ...
     @overload
-    def join(__p1: bytes, __p2: bytes, __p3: Text, *p: _PathType) -> Text: ...
+    def join(__p1: bytes, __p2: bytes, __p3: Text, *p: AnyPath) -> Text: ...
     @overload
-    def join(__p1: bytes, __p2: Text, *p: _PathType) -> Text: ...
+    def join(__p1: bytes, __p2: Text, *p: AnyPath) -> Text: ...
     @overload
-    def join(__p1: Text, *p: _PathType) -> Text: ...
+    def join(__p1: Text, *p: AnyPath) -> Text: ...
 elif sys.version_info >= (3, 6):
     # Mypy complains that the signatures overlap (same for relpath below), but things seem to behave correctly anyway.
```

Comment:
This comment sounds like it's referring to a `# type: ignore` that was since removed, let me remove the comment too.